examples/interactive_example_hr2421.py

```diff
@@ -32,11 +33,11 @@
         arrowprops=dict(arrowstyle="->", connectionstyle='arc,rad=0.5',
             color='green')) )
 annotations.append( sp.plotter.axis.annotate("(only the horizontal\n position matters)",xy=(event1.xdata+10,event1.ydata),xycoords='data') )
-excesslines.append( sp.plotter.axis.vlines(4714,sp.plotter.ymin,sp.plotter.ymax,linestyle='--',color='green') )
+excesslines.append( sp.plotter.axis.vlines(4714,sp.plotter.ymin.value,sp.plotter.ymax.value,linestyle='--',color='green') )
 sp.plotter.refresh()
-sp.plotter.savefig('figures/interactive_example_hr2421_baseline_firstclick.png', bbox_inches=None)
-
+sp.plotter.savefig('interactive_example_hr2421_baseline_firstclick.png', bbox_inches=None)
 event2 = matplotlib.backend_bases.MouseEvent('button_press_event', sp.plotter.axis.figure.canvas,850,280,button=1)
+# event2.xdata and .ydata is None here
```

Comment:
I can reproduce this.  I don't understand it, though.  This is frustrating and we'll have to return to it.... for now, I guess, set it manually:
`event2.xdata = 850`